support-frontend/cloud-formation/cfn.yaml

```diff
@@ -217,8 +217,6 @@ Resources:
       Name: !Sub ${Stack}-${Stage}-${App}
       Port: 9000
       Protocol: HTTP
-      VpcId:
```

Comment:
Was about to ask about this and then I noticed that we specify the VPC twice 🤦‍♂️